### PR TITLE
Enable ETC2 compressed textures. Add asset compressor tool.

### PR DESCRIPTION
**Pull request by MortimerGoro**
…w assets

### CODE REVIEW

**Comment by MortimerGoro** [comment]
I need to also convert the controller model images, just forgot they are in different folders, I'll update the PR, but you can start the review already

**Comment by MortimerGoro** [comment]
Already converted controller images

File: app/src/main/cpp/BrowserWorld.cpp [hunk 1]
```diff
@@ -505,14 +505,16 @@ BrowserWorld::InitializeJava(JNIEnv* aEnv, jobject& aActivity, jobject& aAssetMa
     m.loadingAnimation->LoadModels(m.loader);
     m.rootController->AddNode(m.controllers->GetRoot());
     std::string skyboxPath = VRBrowser::GetActiveEnvironment();
+    std::string extension;
     if (VRBrowser::isOverrideEnvPathEnabled()) {
       std::string storagePath = VRBrowser::GetStorageAbsolutePath(INJECT_SKYBOX_PATH);
       if (std::ifstream(storagePath)) {
         skyboxPath = storagePath;
+        extension = ".jpg";
```
Comment (bluemarvin) [inline app/src/main/cpp/BrowserWorld.cpp]: Would be great to either scan or allow user to specify extension. Definitely a followup, I believe there is already an issue on this.

**Comment by bluemarvin** [comment]
Broken on Focus too.

**Comment by bluemarvin** [comment]
Oops, forgot to update VRB, testing again 😔

**Pull request merged**